cmd/utils/diskusage_openbsd.go

```diff
@@ -14,6 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
+//go:build openbsd
```

Comment:
These go:build annotations seem like duplicates. Right? See the line right below it